### PR TITLE
Show KVM port mappings if they are present.

### PR DESCRIPTION
**Pull request by oponder**

### CODE REVIEW

**Comment by oponder** [comment]
This is what it looks like:

```
$ gsctl show cluster 12345
ID:                            12345
Name:                          My Cluster
Created:                       2017 Nov 08, 14:38 UTC
Organization:                  giantswarm
Kubernetes API endpoint:       https://...
Release version:               n/a
Workers:                       ..
CPU cores in workers:          ..
RAM in worker nodes (GB):      ..
Storage in worker nodes (GB):  ..
Ingress Ports:                 http:  30071
                               https: 30072
```

File: Gopkg.lock [hunk 1]
```diff
@@ -38,10 +38,10 @@
   version = "v2.0.2"
 
 [[projects]]
-  branch = "master"
+  branch = "add-kvm-port-mappings"
```
Comment (oponder) [inline Gopkg.lock]: TODO: change back to master once https://github.com/giantswarm/gsclientgen/pull/25 is merged.

**Comment by marians** [comment]
As for the output format:

```
Ingress Ports:                 http:  30071
                               https: 30072
```

I would prefer if we could stick to the non-hierarchical format we currently have. Like this:

```
Storage in worker nodes (GB):  ..
Ingress port for http:         30071
Ingress port for https:        30072
```

Reason: The output should be usable for simple shell scripting and processing with cut, awk and the likes.

Also please use lowercase p in "port".

**Comment by oponder** [comment]
Ok! adjusted!

**Pull request merged**